### PR TITLE
Adds simple Accelerate integration example

### PR DESCRIPTION
**Pull request by tcapelle**
Simple training script on FashionMNIST using accelerate and multiple GPUs inside a notebook.
- Log only on main process.

### CODE REVIEW

**Comment by tcapelle** [comment]
this one should also be ready

**Comment by morganmcg1** [comment]
looks good, probably want to have a case to avoid .log being called twice between the last train step and the validation step, so that the step counts line up. We don’t highlight that best practice enough and results in user confusion down the line

**Comment by tcapelle** [comment]
How would you do that? I am computing metrics on the train DL as I didn't put any validation DL.
Look at the run here: https://wandb.ai/capecape/accelerate_fmnist?workspace=user-capecape
What one would like is maybe having the epochs line up, but the accuracy at the start is unknown.

**Comment by morganmcg1** [comment]
@tcapelle you'd need a check to see if it was at a validation step or not. It if is, then you could pass `commit=False` for the train metrics

**Comment by tcapelle** [comment]
You mean between these 2: 
image
but it's ugly to pass kwargs to the accelerator:
```python
accelerator.log({"epoch":epoch, "accuracy":accuracy}, log_kwargs={"wandb": {"commit": False}})
```

**Comment by morganmcg1** [comment]
ugly, but the best practice way to do it
